### ISSUE TITLE
Relay opens only with bright theme

### ISSUE THREAD

**Issue by macstar3000** [agronick/Relay#35]
i built and installed relay successfully on my arch linux distro, but when starting, it displays the bright theme by default.

`17:59:13:usaf@AMD>~ echo $XDG_CURRENT_DESKTOP
KDE
`

any chance to fix it?

**Comment by agronick** [comment]
Theres a command line option to override the theme.

**Comment by agronick** [comment]
Or install the pantheon elementary theme. It looks best with that and it
will use it if its there.

**Comment by macstar3000** [comment]
@agronick

thanks running it with a command line option works fine.
however i face another issue now, when i add a irc.freenode.net server with default settings, it can't connect 'no route to host'.
connects fine to other servers.

any idea?

**Comment by agronick** [comment]
Can you ping it or connect with another client?

**Comment by macstar3000** [comment]
@agronick

yes, connecting to irc.freenode.net  with quassel, konversation, etc. works just fine :) 
i notice that it takes longer than to connect to other servers however, while on realy it aborts pretty soon. is there a way to increase server/connection timeout?

**Comment by agronick** [comment]
Yeah that could be it. I"ll look into it.
